dbt/config.py

```diff
@@ -23,6 +24,10 @@
 DEFAULT_USE_COLORS = True
 DEFAULT_PROFILES_DIR = os.path.join(os.path.expanduser('~'), '.dbt')
 
+if os.environ.get('DBT_PROFILES_DIR') is not None:
+    PROFILES_DIR = os.environ.get('DBT_PROFILES_DIR')
+else:
+    PROFILES_DIR = DEFAULT_PROFILES_DIR
```

Comment:
This could just be `PROFILES_DIR = os.environ.get('DBT_PROFILES_DIR', DEFAULT_PROFILES_DIR)`

Reply:
D'oh! Will fix.